build.gradle

```diff
@@ -106,7 +106,7 @@ task cleanIntegrationTests(type: GradleBuild) {
 }
 
 wrapper {
```

Comment:
Why is this `wrapper` needed here anyways? I see you've removed the other 2. Why is this needed?

Reply:
I see that the task options given here only repeat what's already in the GitHub actions file. I'll remove it, thanks.